// protoc -I=./ --python_out=./ gdEng.proto
// syntax="proto3";
package gdEng;

message Device {
  required int32 temperature = 1;
  required int32 ampereConsumption = 2;
  required int32 battery = 3;
  required int64 option = 4;
  required bool power = 5;
  optional string insertTime = 6 [default = "UNINSERTED"];
}

message DeviceData {
  repeated Device device = 1;
}